### PR TITLE
Feature/ciac 9901/jira v3 add delete attachment command

### PR DESCRIPTION
**Pull request by yedidyacohenpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9901

## Description
added a command to delete attachment file from jira issue 

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV3/JiraV3.py [hunk 1]
```diff
@@ -2076,6 +2091,21 @@ def delete_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> Comman
     return CommandResults(readable_output='Issue deleted successfully.')
 
 
+def delete_attachment_file_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    """This command is in charge of deleting an attachment file.
+
+    Args:
+        client (JiraBaseClient): The jira client
```
Comment (samuelFain) [inline Packs/Jira/Integrations/JiraV3/JiraV3.py:2098]:
```suggestion
        client (JiraBaseClient): The jira client.
```

File: Packs/Jira/Integrations/JiraV3/JiraV3.py [hunk 2]
```diff
@@ -2076,6 +2091,21 @@ def delete_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> Comman
...
+        args (Dict[str, str]): The argument supplied by the user
```
Comment (samuelFain) [inline Packs/Jira/Integrations/JiraV3/JiraV3.py:2099]:
```suggestion
        args (Dict[str, str]): The argument supplied by the user.
```

File: Packs/Jira/Integrations/JiraV3/JiraV3_test.py [hunk 3]
```diff
@@ -866,6 +866,24 @@ def test_delete_issue_command(self, mocker):
         assert 'Issue deleted successfully' in command_result.to_context().get('HumanReadable')
 
 
+class TestJiraDeleteAttachmentFileCommand:
+    def test_delete_attachment_file_command(self, mocker: MockerFixture):
+        """
+        Given:
+            - A Jira client
```
Comment (samuelFain) [inline Packs/Jira/Integrations/JiraV3/JiraV3_test.py:873]:
```suggestion
            - A Jira client.
```

File: Packs/Jira/Integrations/JiraV3/README.md [hunk 4]
```diff
@@ -688,6 +688,39 @@ There is no context output for this command.
...
+| --- | --- | --- |
+| attachment_id | The attachment ID. | Required |
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command example
+
+```!jira-issue-delete-file attachment_id=16777```
```
Comment (samuelFain) [inline Packs/Jira/Integrations/JiraV3/README.md:717]: Lets change the `attachment_id` value to "PROJECTKEY" (see other examples from other commands).
Please also change it in `Human Readable Output` accordingly.

**Review by samuelFain — COMMENTED** [review]
Great Job! 🥇

**Review by samuelFain — APPROVED** [review]

File: Packs/Jira/Integrations/JiraV3/JiraV3.yml [hunk 5]
```diff
@@ -806,6 +806,12 @@ script:
       name: issue_key
     description: Delete a comment from an issue.
     name: jira-issue-delete-comment
+  - arguments:
+    - description: The attachment ID.
+      name: attachment_id
+      required: true
+    description: Deletes an attachment from an issue.
```
Comment (jbabazadeh) [inline Packs/Jira/Integrations/JiraV3/JiraV3.yml:813]:
```suggestion
    description: Delete an attachment from an issue.
```

File: Packs/Jira/Integrations/JiraV3/README.md [hunk 6]
```diff
@@ -688,6 +688,39 @@ There is no context output for this command.
 >|---|---|---|---|
 >| https:<span>//</span>api.atlassian.com/ex/jira/1234/rest/api/3/attachment/16505 | dummy.pdf | 16505 | PROJECTKEY-31 |
 
+
+### jira-issue-delete-file
+
+
+***
+Deletes an attachment from an issue.
```
Comment (jbabazadeh) [inline Packs/Jira/Integrations/JiraV3/README.md:696]:
```suggestion
Delete an attachment from an issue.
```

File: Packs/Jira/ReleaseNotes/3_1_11.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Atlassian Jira v3
+
+- Updated the Docker image to: *demisto/btfl-soup:1.0.1.89280*.
+
+- Added a new command ***jira-issue-delete-file*** to delete attached file.
```
Comment (jbabazadeh) [inline Packs/Jira/ReleaseNotes/3_1_11.md:8]:
```suggestion
- Added a new command ***jira-issue-delete-file*** to delete attachment file from an issue.
```

**Review by jbabazadeh — COMMENTED** [review]
Great job!
just small changes.

**Review by jbabazadeh — APPROVED** [review]

**Pull request merged**